src/registrar/utility/email.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for sending emails."""
+
+import boto3
+
+from django.conf import settings
+from django.template.loader import get_template
+
+
+def send_templated_email(template_name: str, to_address: str, context={}):
+    """Send an email built from a template to one email address.
+
+    template_name is relative to the same template context as Django's HTML
+    templates. context gives additional information that the template may use.
+    """
+
+    template = get_template(template_name)
+    email_body = template.render(context=context)
+
+    ses_client = boto3.client(
+        "sesv2",
+        region_name=settings.AWS_REGION,
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
```

Comment:
I'd suggest the use of `max_retries` and `mode` as illustrated [in the docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html#defining-a-retry-configuration-in-a-config-object-for-your-boto3-client). The default is legacy mode with 5 tries, which is probably too many tries for a synchronous request. The docs say that standard mode has 3 tries, which seems reasonable, but I'd use `max_retries` as well, just on the principle that explicit is better than implicit.